packages/analytics/analytics-chart/src/types/chart-types.ts

```diff
@@ -4,6 +4,7 @@ export const chartTypes = [
   'vertical_bar',
   'timeseries_line',
   'timeseries_bar',
+  'choropleth_map',
```

Comment:
This should be the larger Venn diagram that includes all the possible `reportChartTypes` , therefore should include chloropleth as well

Reply:
We need to look into it more tomorrow. Map chart type doesn't result in anything this package is capable of so it shouldn't be here 🤔